bundles/org.openhab.binding.caddx/src/main/java/org/openhab/binding/caddx/internal/handler/ThingHandlerZone.java

```diff
@@ -77,15 +75,17 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         String data = null;
 
         if (command instanceof RefreshType) {
-            if (channelUID.getId().equals(CaddxBindingConstants.ZONE_FAULTED)) {
+            // Refresh only if 2 seconds have passed from the last refresh
+            if (new Date().getTime() - lastRefreshTime > 2000) {
```

Comment:
I think it is a bit more readable if you use `System.currentTimeMillis()` instead of `new Date().getTime()`

Reply:
Changed